neuro_data/static_images/data_schemas.py

```diff
@@ -938,6 +945,11 @@ def fill():
                 StaticMultiDataset.Member().insert1(dict(scan, name = _template.format(**scan)), ignore_extra_fields=True)
 
     def fetch_data(self, key, key_order=None):
+        from neuro_data.static_images.dataset_config import MultiDataset
+        key = (self & key).fetch1()
+        if key in MultiDataset:
```

Comment:
Is it possible to have a valid dataset both in MultiDataset as well as in InputResponse? If the only case where we use MultiDataset is when data source comes from in silico then we may want to assert scan_type == 'in_silico' in StaticScan if fetching data from MultiDataset